modules/account-lib/src/coin/stx/keyPair.ts

```diff
@@ -34,10 +37,11 @@ export class KeyPair extends Secp256k1ExtendedKeyPair {
   }
 
   /**
-   * Build an ECPair from a private key.
+   * Build a keypair from a protocol private key or extended private key.
```

Comment:
Can we pull the extended keypair logic out of this PR? The extended keypair logic was merged separately